test/e2e/network_segmentation_custom_l2udn.go

```diff
@@ -0,0 +1,138 @@
+package e2e
```

Comment:
I was creating `network_segmentation_default_network_annotation.go` since I am testing the pod's default-network annotation, does it make sense to have two different test suites or should I follow your file name and Describe name ?

Reply:
I don't mind having more than one suite as these are not strictly depended. Up to you!